src/commands/run.js

```diff
@@ -43,12 +44,18 @@ export async function run({ taskName, options }) {
   if (logger instanceof InteractiveLogger) logger.clearTasks()
 
   const failedTasks = tasks.allFailedTasks()
-
   if (failedTasks.length > 0) {
     logger.fail(`Failed tasks: ${failedTasks.join(', ')}`)
   }
 
-  if (Object.entries(tasks.allTasks).every(([, task]) => task.status === 'success:lazy')) {
-    logger.log('\n' + rainbow('>>> MAXIMUM LAZY'))
+  const stats = tasks.getTaskStats()
+  logger.log(`\n\t   Tasks:\t${kleur.green(stats.successful)} successful, ${stats.allTasks} Total`)
```

Comment:
Doesn't look like we support fixed width outputs. Could improve this if we change the logger we use.

Reply:
Added `padString` function to help with aligning things.